app/resources/view.html.template

```diff
@@ -17,15 +17,22 @@
   {{person.full_name}} - {{block.super}}
 {% endblock %}
 
-{% if params.query or params.given_name or params.family_name %}
-  {% block logo %}
+{% block logo %}
+  {% if params.query_name or params.given_name or params.family_name %}
```

Comment:
That sounds weird. Do you have any idea why? If you use {{block.super}}, shouldn't it be equivalent to the logic in original order?

If you have no idea then that's fine because it's anyway fixed, but I'm just curious.